tests/SslCertificateFromStringTest.php

```diff
@@ -0,0 +1,216 @@
+<?php
+
+namespace Spatie\SslCertificate\Test;
+
+use Carbon\Carbon;
+use PHPUnit\Framework\TestCase;
+use Spatie\Snapshots\MatchesSnapshots;
+use Spatie\SslCertificate\SslCertificate;
+
+class SslCertificateFromStringTest extends TestCase
+{
+    use MatchesSnapshots;
+
+    /** @var SslCertificate */
```

Comment:
Use the fully qualified class name in the comment.

Reply:
Check, changed it also for the class I used to copy from :)